.wp-env.json

```diff
@@ -2,10 +2,11 @@
   "core": null,
   "plugins": [
     ".",
+    "./plugins/optimization-detective",
```

Comment:
This has to appear before `image-prioritizer` or else an error occurs since it is a dependency.

Reply:
please add an inline comment explaining this so it doesn't get moved

Reply:
Unfortunately inline comments are not allowed in JSON

Reply:
We'd have to hack it with something like this:

```json
{
  "core": null,
  "// Notice": "The optimization-detective plugin must be listed first since it is a dependency.",
  "plugins": [
    "./plugins/optimization-detective",
    "./plugins/auto-sizes",
    "./plugins/dominant-color-images",
    "./plugins/embed-optimizer",
    "./plugins/image-prioritizer",
    "./plugins/performance-lab",
    "./plugins/speculation-rules",
    "./plugins/webp-uploads"
  ],
  "env": {
    "tests": {
      "config": {
        "FS_METHOD": "direct"
      },
      "mappings": {
        "/wp-content/plugins/performance": "."
      }
    }
  }
}
```

Reply:
Actually, that doesn't work. I get an error:

```
✖ Invalid /home/westonruter/repos/performance/.wp-env.override.json: "// Note" is not a configuration option.
```

Reply:
Yeah I don‘t think we need a clever comment here. If you try to move it, there will be an error anyway.

Reply:
right, sorry - I didn't realize this was in JSON when suggesting a comment.